### ISSUE TITLE
SUPER ROBOT WARS T

### ISSUE THREAD

**Issue by GoffyDude** [Ryujinx/Ryujinx-Games-List#1029]
#### Current on `master` : 1.0.4398

Crashes on launch.

```
Emulation CurrentDomain_UnhandledException: Unhandled exception caught: Ryujinx.HLE.Exceptions.ServiceNotImplementedException: Ryujinx.HLE.HOS.Services.Ns.IAddOnContentManager: 8
```
#### Outstanding Issues:

https://github.com/Ryujinx/Ryujinx/issues/636

#### Log file :

[SRWT.zip](https://github.com/Ryujinx/Ryujinx-Games-List/files/4565987/SRWT.zip)

**Comment by dawnbomb** [comment]
yeah, this used to run, theres videos even a few builds ago, now it crashes on boot.

**Comment by gdkchan** [comment]
I believe the people running it had ignore missing services enabled on the settings to bypass that.

**Comment by dawnbomb** [comment]
Yes, im saying it doesn't boot when even using ignore missing services.

**Comment by EmulationFanatic** [comment]
Needs update

**Comment by EmulationFanatic** [comment]
Updated

**Comment by squall1027** [comment]
It can run T, but can't save & load after save

**Comment by wang59prince** [comment]
this used to run but crashed sometime while saving, and now, it crashed on boot. ver 6794

**Comment by EmulationFanatic** [comment]
@wang59prince working fine here without any problems. 
![image](https://user-images.githubusercontent.com/62343878/112456119-0a86ac00-8d20-11eb-8068-d27945b3c4f3.png)
![image](https://user-images.githubusercontent.com/62343878/112456344-4cafed80-8d20-11eb-8a91-bcc7bcd36753.png)

**Comment by squall1027** [comment]
> @ wang59prince在這里工作正常，沒有任何問題。
> ![圖像](https://user-images.githubusercontent.com/62343878/112456119-0a86ac00-8d20-11eb-8068-d27945b3c4f3.png)
> ![圖像](https://user-images.githubusercontent.com/62343878/112456344-4cafed80-8d20-11eb-8a91-bcc7bcd36753.png)

Open game ->  load / new save  -> Save Game -> reload game UI (Not Ryujinx reload Function) -> Load Save ( crash )
